lib/command.py

```diff
@@ -1,108 +1,113 @@
+                                            "e.g. 'say Hello'")
+        self.game_state.tell_player(player, "  look           - Examines the "
+                                            "surroundings, e.g. 'look'")
+        self.game_state.tell_player(player, "  go <exit>      - Moves through the exit "
+                                            "specified, e.g. 'go outside'")
+
+    def look(self, player: Player, params=None):
+        # store the player's current room
+        current_location = rooms[player.location]
 
-        # update the player's current room to the one the exit leads to
-        player.location = current_location["exits"][ex]
+        # send the player back the description of their current room
+        self.game_state.tell_player(player, current_location['description'])
+
+        players_here = []
+        # go through every player in the game
+        for other_player in self.game_state.list_players():
+            # if they're in the same room as the player
+            if other_player.location == player.location:
+                # ... and they have a name to be shown
+                if other_player.name:
+                    # add their name to the list
+                    players_here.append(other_player.name)
+
+        # send player a message containing the list of players in the room
+        self.game_state.tell_player(player, "Players here: {}".format(", ".join(players_here)))
+        # send player a message containing the list of exits from this room
+        self.game_state.tell_player(player, "Exits are: {}".format(", ".join(current_location["exits"])))
+
+    def go(self, player: Player, params):
+        # store the exit name
+        ex = params.lower()
+
+        # store the player's current room
         current_location = rooms[player.location]
 
-        # go through all the players in the game
-        for other_player in game.players.values():
-            # if player is in the same (new) room and isn't the player
-            # sending the command
-            if other_player.location == player.location and other_player.uuid != player.uuid:
-                # send them a message telling them that the player
-                # entered the room
-                tell_player(other_player, f"{player.name} arrived via exit '{ex}'")
-
-        # send the player a message telling them where they are now
-        tell_player(player, f"You arrive at '{player.location}'")
-        tell_player(player, rooms[player.location]["description"])
-
-    # the specified exit wasn't found in the current room
-    else:
-        # send back an 'unknown exit' message
-        tell_player(player, f"Unknown exit '{ex}'")
+        # if the specified exit is found in the room's exits list
+        if ex in current_location["exits"]:
+
+            # go through all the players in the game
+            for other_player in self.game_state.list_players():
```

Comment:
Again, I would love if there was some way we could more cleanly expose these methods.